Upgrade.md

```diff
@@ -0,0 +1,55 @@
+# Description
+
+This guide will hopefully provide the help and tools to upgrade Substrate in the future.
+
+This is a guide based on updating from `0.9.25` to `0.9.30` and the issues it caused can be seen [here](https://github.com/digicatapult/dscp-node/pull/91/files)
+
+# Prerequisities
+
+Create an **integration** branch which all other branches will be PR'd into. This reduces git issues like having to use `git reset`.
+
+You will need to run `rustup` which is [documented here](https://github.com/digicatapult/dscp-node/blob/main/README.md).
+
+# Upgrade Substrate
+
+This [diener tool](https://crates.io/crates/diener) can be used to update the toml versions fairly painlesslly by upgrading each `cargo.toml` given a specific branch/path.
+
+# Build the Node
+
+To build the Node:
+
+```bash
+cargo build --release
+```
+
+# Pallets
+
+After the version updates have been made, depending on the length of time between version releases, there may be a **lot** of changes.
+
+If a file has errors run `cargo check` and it will only check that one file.
```

Comment:
`cargo check` will actually type check the whole workspace. Not sure what you're getting at here